mods/creative/init.lua

```diff
@@ -55,25 +55,27 @@ if minetest.is_creative_enabled("") then
 	local digtime = 42
 	local caps = {times = {digtime, digtime, digtime}, uses = 0, maxlevel = 256}
```

Comment:
Why not move these inside the function to further limit their scope (along with the comment) rather than having them as upvalues?

Reply:
Done. Thanks for pointing that out.